wlanpi_core/api/api_v1/endpoints/network_api.py

```diff
@@ -3,7 +3,9 @@
 
 from fastapi import APIRouter, Depends, Response
 
+
 from wlanpi_core.core.auth import verify_jwt_token
+from wlanpi_core.constants import API_DEFAULT_TIMEOUT
```

Comment:
Hey @Orbitix, looks like the import order is not quite right. Should be in alphabetical order which means `constants import API_...` would be before `core.auth import verify...`.

Reply:
fixed in commit [1a04478](https://github.com/WLAN-Pi/wlanpi-core/pull/56/commits/1a04478b34aaa32f56cec66b6e7b98f5d7622528), style police no longer fails